Mirror/Runtime/NetworkTime.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using UnityEngine;
+
+namespace Mirror
+{
+    // calculates synchronized time and rtt
+    public class NetworkTime 
+    {
+        // some arbitrary point in time where time started
+        static readonly DateTime epoch = new DateTime(2018, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
Do you know how much of a problem Jan 01 1970 would be here?
If it's just off by a nanosecond when casting to float then it might be worth having that unix standard here